test/NNPDE_tests.jl

```diff
@@ -247,20 +288,37 @@ dim = length(domains)
 expr_pde_loss_function = NeuralPDE.build_loss_function(eq,indvars,depvars)
 expr_bc_loss_functions = [NeuralPDE.build_loss_function(bc,indvars,depvars) for bc in bcs]
 
-train_sets = NeuralPDE.generate_training_sets(domains,discretization,bcs,indvars,depvars)
+train_sets = NeuralPDE.generate_training_sets(domains,dx,bcs,indvars,depvars)
+
+train_domain_set,train_bound_set,train_set= train_sets
+
+# equation/system of equations
+isuinplace = chain.layers[end].out == 1
 
-train_domain_set, train_bound_set = train_sets
+phi = discretization.phi
+autodiff = discretization.autodiff
+derivative = discretization.derivative
 
-pde_loss_function = NeuralPDE.get_loss_function(eval(expr_pde_loss_function),train_domain_set)
-bc_loss_function = NeuralPDE.get_loss_function(eval.(expr_bc_loss_functions),train_bound_set)
+pde_loss_function = NeuralPDE.get_loss_function(eval(expr_pde_loss_function),
+                                      train_domain_set,
+                                      phi,
+                                      derivative,
+                                      training_strategies)
+bc_loss_function = NeuralPDE.get_loss_function(eval.(expr_bc_loss_functions),
+                                     train_bound_set,
+                                     phi,
+                                     derivative,
+                                     training_strategies)
 
-function loss_function(θ, phi, derivative)
-    return pde_loss_function(θ, phi, derivative) + bc_loss_function(θ, phi, derivative)
+function loss_function(θ)
+    return pde_loss_function(θ) + bc_loss_function(θ)
 end
 
-prob = GalacticOptim.OptimizationProblem(loss_function, zeros(dim))
+initθ = discretization.initθ
+res = DiffEqFlux.sciml_train(loss_function, initθ, Flux.ADAM(0.1); cb = cb, maxiters=1000)
```

Comment:
this just to test sciml_train? Note that will be deprecated for GalacticOptim once it's ready